airbyte-integrations/connectors/source-opsgenie/source_opsgenie/schemas/users.json

```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": ["null", "string"]
```

Comment:
Use `number` as type instead of `string`

Reply:
As above, id is a UUID:

```
{
    "totalCount": 200,
    "data": [
        ...
        {
            "blocked": false,
            "verified": true,
            "id": "a9d98683-7ddf-40da-a68b-2eadb54f16e8",
            "username": "adam.schmidt@zip.co",
            "fullName": "Adam Schmidt",
            "role": {
                "id": "Admin",
                "name": "Admin"
            },
            "timeZone": "Australia/Sydney",
            "locale": "en_AU",
            "userAddress": {
                "country": "",
                "state": "",
                "city": "",
                "line": "",
                "zipCode": ""
            },
            "createdAt": "2019-04-17T23:25:46.48Z"
        },
        ...
    ],
    "paging": {
        "next": "https://api.opsgenie.com/v2/users?limit=100&offset=100&order=ASC&sort=username",
        "first": "https://api.opsgenie.com/v2/users?limit=100&offset=0&order=ASC&sort=username",
        "last": "https://api.opsgenie.com/v2/users?limit=100&offset=100&order=ASC&sort=username"
    },
    "took": 0.122,
    "requestId": "43d671ea-ef33-40b8-9780-bd44ab9205d0"
}
```